docs/pages/admin-guides/access-controls/sso/sso.mdx

```diff
@@ -615,6 +615,59 @@ If something is not working, we recommend to:
 
 - Double-check the host names, tokens and TCP ports in a connector definition.
 
+## Identifier-first login
+
+Identifier-first login is a feature that simplifies the login experience in clusters with
+multiple authentication connectors. When identifier-first login is enabled, the initial
+login screen prompts the user for their username. Teleport will use this username to identify the user,
+and then only display the auth connector(s) that are relevant to them.
+
+This feature can be enabled on a cluster by configuring any one or more SAML, OIDC, or
+GitHub auth connector with a `spec.user_matchers` field, which takes a set of one or more
+user matchers for usernames. A user matcher is a string matcher that can either be a specific
+username to match, or a glob pattern using wildcard characters (`*`). An auth connector
+will be mapped to a user if one or more of its `user_matchers` matches their username.
+
+```yaml
+kind: saml
+version: v2
+metadata:
+  name: okta
+spec:
+  user_matchers: ['joe', '*@example.com']
+```
+
+Once enabled, any users without an auth connector mapped to them will not be able to see
+any auth connectors on the login page. For this reason, it is recommended to configure
+one or more fallback connectors. A user matcher of `*` is a special case that marks the
+connector as a fallback, which will only be presented to users who don't match any other connector.
+
+#### Examples
+
+Match users with a username that ends in "@example.com"
```

Comment:
I would add a colon at the end of each sentence that introduces an example snippet